src/main/java/org/threeten/extra/chrono/InternationalFixedChronology.java

```diff
@@ -0,0 +1,484 @@
+ */
+
+package org.threeten.extra.chrono;
+
+import java.io.Serializable;
+
+import java.time.Clock;
+import java.time.DateTimeException;
+import java.time.ZoneId;
+
+import java.time.chrono.AbstractChronology;
+import java.time.chrono.Era;
+
+import java.time.temporal.ChronoField;
+import java.time.temporal.TemporalAccessor;
+import java.time.temporal.ValueRange;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * The International Fixed calendar system.
+ * <p>
+ * This chronology defines the rules of the International Fixed calendar system.
+ * It shares the leap year rule with the Gregorian calendar.
+ * Dates are aligned such that {@code 0001-01-01 (International Fixed)} is {@code 0001-01-01 (ISO)}.
+ * <p>
+ * This class is not proleptic.  It implements accepts only years greater or equal to 1.
+ * <p>
+ * This class implements a calendar where January 1st is the start of the year.
+ * <p>
+ * The fields are defined as follows:
+ * <ul>
+ * <li>era - There is only one era, the current 'Common Era' (CE).
+ * <li>year-of-era - The year-of-era for the current era increases uniformly from the epoch at year zero.
+ * <li>proleptic-year - The proleptic year is the same as the year-of-era for the current era.
+ * <li>month-of-year - There are 13 months in an International Fixed year, numbered from 1 to 13.
+ * <li>day-of-month - There are 28 days in an International Fixed month, numbered from 1 to 28.
+ * <li>day-of-year - There are 365 days in a standard International Fixed year and 366 days in a leap year.
+ *  The days are numbered accordingly.
+ * <li>leap-year - Leap years occur every 4 years, but skips 3 out of four centuries, i.e. when the century is not divisible by 400.
+ *  This is the same rule in use for the Gregorian calendar.
+ * <li>Week day - every month starts on a Sunday.  Leap-day and year-day are considered month-less Sundays.
+ * </ul>
+ *
+ * <h3>Implementation Requirements</h3>
+ * This class is immutable and thread-safe.
+ */
+public final class InternationalFixedChronology extends AbstractChronology implements Serializable {
+    /**
+     * Singleton instance for the International fixed chronology.
+     */
+    public static final InternationalFixedChronology INSTANCE = new InternationalFixedChronology();
+    /**
+     * Serialization version UID.
+     */
+    private static final long serialVersionUID = -1050847936470712250L;
+    /**
+     * Standard 7-day week.
+     */
+    static final int DAYS_IN_WEEK = 7;
+    /**
+     * In all months, there are 4 complete weeks.
+     */
+    static final int WEEKS_IN_MONTH = 4;
+    /**
+     * There are 13 months in a year.
+     */
+    static final int MONTHS_IN_YEAR = 13;
+    /**
+     * There are 4 weeks of 7 days, or 28 total days in a month.
+     */
+    static final int DAYS_IN_MONTH = WEEKS_IN_MONTH * DAYS_IN_WEEK;
+    /**
+     * Range of day-of-week.
+     */
+    static final ValueRange DAY_OF_WEEK_RANGE = ValueRange.of(1, DAYS_IN_WEEK);
```

Comment:
Since year-day and leap-day aren't Sunday, you need to account for whatever special value you use - I recommend `0` for this purpose here: `ValueRange.of(0, 1, DAYS_IN_WEEK)`